app/uk/gov/hmrc/customs/notification/logging/LoggingHelper.scala

```diff
@@ -107,8 +107,19 @@ object LoggingHelper {
         formatOptional("mrn", has.maybeMrn)
       case _ => ""
     }
+    def entryNumber = rm match {
```

Comment:
The file could be reformatted to have line breaks between defs, to make easier to read.